File: VB/E4924/Models/Person.vb
Imports System.ComponentModel.DataAnnotations

Public Class Person
    <Required>
    Public Property ID As Integer
    <Required>
    Public Property Name As String
    Public Property CompanyName As String
    <Custom(MinAge:=18)>
    Public Property BirthDate As DateTime
    Public Property CheckAge As Boolean
End Class

File: CS/E4924/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace E4924.Models
{
    public class Person
    {
        [Required]
        public int ID { set; get; }
        [Required]
        public string Name { set; get; }
        public string CompanyName { set; get; }
        [Custom(MinAge = 18)]
        public DateTime BirthDate { set; get; }
        public bool CheckAge { set; get; }
    }
}